examples/ros/src/smarts_ros/scripts/ros_driver.py

```diff
@@ -121,6 +125,8 @@ def setup_ros(
             assert target_freq > 0.0
             self._target_freq = target_freq
 
+        log_everything_to_ROS(level=logging.WARNING)
```

Comment:
if I want to log to ros in the bm code, is this all i need to call in the init of the agent? or would this need to be called in act

Reply:
No, actually you shouldn't have to do anything.  This line here will apply to all existing and all future loggers too, including your agent's.  You can look at my current agent for an example of how I configure it's logging.  Once this PR gets merged, those log messages will show up both in the Smarts logfile and in the rospy stream to stdout.